airflow/executors/celery_executor.py

```diff
@@ -82,7 +82,7 @@ def execute_async(self, key, command,
         self.log.info("[celery] queuing {key} through celery, "
                       "queue={queue}".format(**locals()))
         self.tasks[key] = execute_command.apply_async(
-            args=command, queue=queue)
+            args=[command], queue=queue)
```

Comment:
I'm confused here - what type is `command` - is it a  string (shouldn't be anymore?) or a list?

If its a list doesn't this make the args passed to execute_command be `[['airflow', 'run', ...]]`. Is that what it needs to be?

Reply:
Oh, this worked before because Celery is doing the equivalent of `*args` at some point, so `execute_command('true')` was what the tests called. Right.

Might be clearer to use `kwargs={'command'=command}` if celery supports it?